src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -316,6 +316,10 @@ public function prepare(Request $request)
                     $headers->set('Content-Length', $length);
                 }
             }
+
+            if (!$headers->has('Content-Language')) {
+                $headers->set('Content-Language', $request->getLocale());
```

Comment:
can't this lead to HTTP cache poisoning?

Reply:
Indeed, if this header is set, it must be added to ˋVary`.

Reply:
Also, I would not set this headers if there is not at least one value in  ˋavailable_locales`. To not change the behavior of existing apps.